webapp/src/utils.ts

```diff
@@ -108,19 +108,36 @@ export function isCallsPopOut(): boolean {
     try {
         return window.opener && window.opener.callsClient;
     } catch (err) {
-        logErr(err);
+        // This can happen if the MM window already has an opener on a different origin (e.g. user clicked a link on a calendar app to open MM).
+        // In this case, we know we are not in the expanded view so we can directly return window.callsClient.
+        logWarn(err);
```

Comment:
We are lowering the level to warning, but we'd still get quite the spam in such a case since this is used by a few components and selectors. I'll see if there's a way to detect the specific case and avoid logging, but I'm not sure.

Reply:
Something like: ?
```suggestion
        if (e.name !== 'SecurityError') {
            // This is likely not a cross-origin error.
            logErr(err);
  }
```